### PR TITLE
Remove dependence on unmaintained satori UUID library.

### PR DESCRIPTION
**Pull request by fasaxc**
## Description

* Switch to Google's library as used by k8s.
* Rev tchap/go-patricia to avoid indirect dependency.
 
## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
Remove dependence on unmaintained UUID library.
```

### CODE REVIEW

**Review by neiljerram — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
